server/test/expand.go

```diff
@@ -258,30 +365,35 @@ func TestExpandQuery(t *testing.T, datastore storage.OpenFGADatastore) {
 			},
 		},
 		{
-			name: "tuple to userset implicit",
-			typeDefinitions: []*openfgapb.TypeDefinition{
-				{
-					Type: "repo",
-					Relations: map[string]*openfgapb.Userset{
-						"admin": {
-							Userset: &openfgapb.Userset_TupleToUserset{
-								TupleToUserset: &openfgapb.TupleToUserset{
-									Tupleset: &openfgapb.ObjectRelation{
-										Relation: "manager",
-									},
-									ComputedUserset: &openfgapb.ObjectRelation{
-										Relation: "repo_admin",
+			name: "1.0_tuple_to_userset_II",
+			model: &openfgapb.AuthorizationModel{
+				Id:            ulid.Make().String(),
+				SchemaVersion: typesystem.SchemaVersion1_0,
+				TypeDefinitions: []*openfgapb.TypeDefinition{
+					{
+						Type: "repo",
+						Relations: map[string]*openfgapb.Userset{
+							"admin": {
+								Userset: &openfgapb.Userset_TupleToUserset{
+									TupleToUserset: &openfgapb.TupleToUserset{
+										Tupleset: &openfgapb.ObjectRelation{
```

Comment:
Here and elsewhere, let's please use the `typesystem` helpers. 

`typesystem.TupleToUserset("manager", "repo_admin")`

and the others (where applicable) `typesystem.ComputedUserset(...)` and `typesystem.This()`